### DISCUSSION TITLE
Repo Description and Tags

### DISCUSSION THREAD

**Discussion by mjakeman** [gircore/gir.core#142] [Ideas]
Looking at our repository description, we don't explicitly mention GTK anywhere. While "GObject" is more correct, it doesn't mean all that much to the average C# dev.

Maybe

```
"A C# wrapper for GObject based libraries providing a C# friendly API surface"
```

could become

```
"A C#-friendly wrapper for the GTK UI Toolkit, GStreamer Multimedia framework, and other GObject based libraries."
```

That way we're more likely to come up in searches for GTK and GStreamer, and it's clear those are major parts of the wrapper.

Thoughts? @badcel @na2axl

Also, I've noticed we don't actually mention C# in our repository tags 😅. I've gone ahead and added `csharp` on the end.

**Comment by badcel** [comment]
It is in the repository tags. I hope that this results in being found.

For me it is okay to mention other projects but I would like to emphasis on GObject. Gstreamer is potentially as big as Gtk but has a completely different audience.

For me the goal of this project is to provide a foundation for GOject based libraries with some out of the box libs like gtk etc.. We are / will be more than just bindings. We will allow to create arbitrary bindings for every GObect based library.

For me this is a lot more than "Gtk, Gstreamer and others".
